pkg/ui/workspaces/db-console/src/views/databases/databaseTablePage/redux.ts

```diff
@@ -71,8 +70,8 @@ export const mapStateToProps = createSelector(
     isTenant,
     hasAdminRole,
   ): DatabaseTablePageData => {
-    const details = tableDetails[generateTableID(database, table)];
-    const indexStats = indexUsageStats[generateTableID(database, table)];
+    const details = tableDetails[util.generateTableID(database, table)];
+    const indexStats = indexUsageStats[util.generateTableID(database, table)];
```

Comment:
nit: This mapStateToProps is still using createSelector if you want to clean that up in this PR.

Reply:
I've [refactored that](https://github.com/cockroachdb/cockroach/pull/104180/files#diff-4167077d5580805359d4e6040bb481c1fc11edb4287eb8cff040e01d07051fd4) in the follow-up table details PR